### PR TITLE
doc: prepare new release 3.1.2

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Ref: #3622 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [X] Manual, quality tests NOT RELEVANT

### CODE REVIEW

**Review by raphj — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Some modification are necessary (work in progress).

File: changelog.md [hunk 1]
```diff
@@ -1,4 +1,25 @@
 
+## 3.1.2 / 2020-09-25
+
+### New Features
+
+- Start showing members in the mention suggestions right away
+- Keep only chosen TLMs in notification panel.
```
Comment (raphj) [inline changelog.md:7]:
```suggestion
- Disable some notifications types by default to unclutter the notification wall.
```

File: changelog.md [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+- Mention: [#3631](https://github.com/tracim/tracim/issues/3631)
+
+## Known Issues
+
+- Editing out a mention does not remove its associated metadata, which leads to an error when saving the document being edited in app text documents (issue [#3640](https://github.com/tracim/tracim/issues/3640)
+- Clicking on a notification sometimes does not correctly open the associated content (issue [#3578](https://github.com/tracim/tracim/issues/3578))
+
+### Other Changes
+
+- New parameters `web.notifications.excluded` added in `development.ini.sample` to define what you want to exclude in notifications panel. You need to update your configuration file manually on existing installations.
```
Comment (raphj) [inline changelog.md:20]:
```suggestion
- New parameter `web.notifications.excluded` added in `development.ini.sample` to define what you want to exclude in notifications panel. There is no need to update your configuration file on existing installations if you want to keep the default set of excluded notifications.
```

**Review by raphj — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md:20]: This is a good idea to explain what is necessary like this. I very like this approach.

**Pull request merged**